### PR TITLE
Enable chrome os download

### PR DESCRIPTION
**Pull request by rudibravo**
@Rise-Vision/apps @Rise-Vision/delivery please review

### CODE REVIEW

File: web/partials/displays/display-fields.html [hunk 1]
```diff
@@ -230,7 +230,7 @@
             </td>
           </tr>
 
-          <tr class="table-body__row" ng-if="!playerProFactory.isCROS(display)">
+          <tr class="table-body__row" ng-if="!playerProFactory.isCROSLegacy(display) && !playerProFactory.isChromeOSPlayer(display)">
```
Comment (fjvallarino) [inline web/partials/displays/display-fields.html]: @rudibravo just to be sure: the new CROS player does not support Offline Play, right?

File: web/partials/displays/display-add-modal.html [hunk 2]
```diff
@@ -148,7 +146,7 @@ <h5>
       <div ng-show="currentTab == 'chromeos'" class="prototype-download-content row u_margin-md">
         <div class="col-xs-6 col-xs-offset-3 text-center">
           <!-- LINK PENDING FROM DELIVERY -->
```
Comment (fjvallarino) [inline web/partials/displays/display-add-modal.html]: @rudibravo I guess this line can be removed now

File: web/partials/displays/display-fields.html (hunk 1 above)
Comment (rudibravo) [reply web/partials/displays/display-fields.html]: @fjvallarino that's right

File: web/partials/displays/display-add-modal.html (hunk 2 above)
Comment (rudibravo) [reply web/partials/displays/display-add-modal.html]: @fjvallarino good point, I missed it.

File: test/unit/displays/directives/dtv-screenshot.tests.js [hunk 3]
```diff
@@ -83,6 +87,9 @@ describe('directive: screenshot', function() {
       expect($scope.screenshotState({ playerVersion: '2018', os: 'cros-x64' })).to.equal('os-not-supported');
       expect($scope.screenshotState({ playerVersion: '2018', os: 'Microsoft' })).to.equal('upgrade-player');
       expect($scope.screenshotState({ playerVersion: '2018' })).to.equal('upgrade-player');
+      isChromeOS = true;
+      expect($scope.screenshotState({ playerVersion: '2018' })).to.equal('chromeos-soon');
+      isChromeOS = false;
```
Comment (andrecardoso) [inline test/unit/displays/directives/dtv-screenshot.tests.js]: This is weird, but I suppose there's a good reason for changing this variable.

Comment (rudibravo) [reply test/unit/displays/directives/dtv-screenshot.tests.js]: @andrecardoso it's used to mock if the display is using our chrome os player.

**Pull request merged**